### PR TITLE
Add columns adjustments

### PR DESCRIPTION
**Pull request by tina-pina**
Test URLs:
- Before: https://main--siemens-press--groundfoghub.hlx.page/press-materials-siemens-ag
- After: https://add-columns-adjustments--siemens-press--groundfoghub.hlx.page/press-materials-siemens-ag

### CODE REVIEW

File: blocks/columns/columns.js [hunk 1]
```diff
@@ -13,6 +13,29 @@ export default function decorate(block) {
...
+
+        headlineElement.classList.add('columns-heading');
+      } else if (col.firstElementChild?.tagName === 'H1'
+        || col.firstElementChild?.tagName === 'H2'
+        || col.firstElementChild?.tagName === 'H3'
+      ) {
+        col.firstElementChild.classList.add('columns-heading');
+      }
+      const colChildren = [...col.children];
+      colChildren.shift();
```
Comment (seuck) [inline blocks/columns/columns.js:36]: Please add a comment why you can safely get rid of first child.

File: blocks/columns/columns.css [hunk 2]
```diff
@@ -1,10 +1,13 @@
+/* stylelint-disable */
```
[the remark this answers is not in the document]
Comment (tina-pina) [reply blocks/columns/columns.css:1]: @seuck Ok but than i can use it in the same file under the project CSS for the Siemens.com related styles right?

[the remark this answers is not in the document]
Comment (seuck) [reply blocks/columns/columns.css:1]: You can add the disable line just before the CSS rules you do not want to be linted

File: blocks/columns/columns.js (hunk 1 above)
Comment (tina-pina) [reply blocks/columns/columns.js:55]: added a comment

File: blocks/columns/columns.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (tina-pina) [reply blocks/columns/columns.css]: updated it @seuck

File: blocks/columns/columns.js [hunk 3]
```diff
@@ -1,10 +1,14 @@
+import { getHeadingOptions } from '../../scripts/utils.js';
```
[the remark this answers is not in the document]
Comment (tina-pina) [reply blocks/columns/columns.js]: updated @seuck

**Pull request merged**